index.html

```diff
@@ -557,6 +571,96 @@ <h4>
             implementations. [[WebIDL]]
           </p>
         </section>
+        <section>
+          <h4>
+            <dfn>PerformanceTimingConfidence</dfn> interface
+          </h4>
+          <pre class='idl'>
+          [Exposed=Window]
+          interface PerformanceTimingConfidence {
+              readonly attribute double randomizedTriggerRate;
+              readonly attribute PerformanceTimingConfidenceValue value;
+              object toJSON();
+          };
+          </pre>
+          <dl data-dfn-for='PerformanceTimingConfidence'>
+            <dt>
+              <dfn>randomizedTriggerRate</dfn>
+            </dt>
+            <dd>
+              <p>
+                This attribute must return a decimal number between 0 and 1
+                indicating how often noise is applied. It represents the
+                probability that value will be randomly replaced with a fake
+                confidence value instead of the real one.
+              </p>
+            </dd>
+            <dt>
+              <dfn>value</dfn>
+            </dt>
+            <dd>
+              <p>
+                This attribute must return a {{PerformanceTimingConfidenceValue}}
+                indicating the confidence value for the timing object.
+              </p>
+            </dd>
+            <dt>
+              <dfn>toJSON()</dfn>
+            </dt>
+            <dd>
+              <p>
+                This method runs the [=default toJSON steps=] for [=this=].
+              </p>
+            </dd>
+          </dl>
+        </section>
+
+        <section>
+          <h4>
+            <dfn>PerformanceTimingConfidenceValue</dfn> enum
+          </h4>
+          <pre class='idl'>
+          enum PerformanceTimingConfidenceValue {
+            "high",
+            "low"
+          };
+          </pre>
+          <p>
+            The values are defined as follows:
+          </p>
+          <dl data-dfn-for='PerformanceTimingConfidenceValue'>
+            <dt>
+              <dfn>high</dfn>
+            </dt>
+            <dd>
+              The user agent considers the navigation metrics to be
+              representative on the current user's device.
+            </dd>
+            <dt>
+              <dfn>low</dfn>
+            </dt>
+            <dd>
+              The navigation metrics may not be representative of the current
+              user's device. The user agent may consider the state of the
+              machine, or user configuration.
+            </dd>
+          </dl>
+          <p class="note">
```

Comment:
Maybe instead of defining these things in a note, we can define an internal "navigation confidence" boolean that is [implementation defined](https://infra.spec.whatwg.org/#implementation-defined), a "random rate" that's also implementation-defined, and then reflect both of these (with an algorithm that defines how the random rate is applied) in the web exposed attributes?